markdown/blog/2025-march-summary.md

```diff
@@ -0,0 +1,101 @@
+
+
+### AsyncAPI Meetup Section
+We plan to introduce a meetups section under the AsyncAPI Conference umbrella, a platform enabling community members to connect, share knowledge, and discuss topics related to AsyncAPI outside of the main conference events. The goal is to foster collaboration and knowledge-sharing within the AsyncAPI ecosystem.
+
+As part of preparing the section, we will have an open discussion on the guidelines and rules. Additionally, we have an open issue with the design for implementing the page for the conference website. I encourage everyone to share their thoughts and [review the AsyncAPI Meetups Section discussion](https://github.com/asyncapi/conference-website/discussions/604) and [the AsyncAPI Meetups Page issue](https://github.com/asyncapi/conference-website/issues/644).
+
+## Spec x Tooling
+
+We are excited to welcome a new project, [Neuroglia.AsyncAPI](https://github.com/asyncapi/net-sdk), a .NET SDK for visualizing and interacting with AsyncAPI documents donated by [Charles d'Avernas](https://www.linkedin.com/in/charles-d-avernas-40836a13/). The project also supports a code-first documentation generator, fluent builders for intuitive and expressive API definitions, and fluent validators to ensure compliance and correctness. All the components also support the AsyncAPI v3. [Be sure to try out Neuroglia.AsyncAPI for your documents](https://github.com/asyncapi/net-sdk), especially if you are a .NET developer!
+
+### RAML Schema Parser
+
+Currently, we do not have active maintainers for the RAML schema parser, a standard for RAML data types. If any community member depends on this parser and is interested in maintaining the project or has any thoughts on this issue, [please reach out or share them in the open discussion about the RAML Parser](https://github.com/asyncapi/raml-dt-schema-parser/issues/255).
+
+
+## AsyncAPI Governance Board 
+ We are still discussing the new AsyncAPI Governance Board, which will take effect later this year. Part of the changes this will bring will be to replace the single Executive Director position and have the option of Ambassadors becoming TSC members. You still have a chance to share your thoughts and views; please do so [under the introduction of the governance board discussion](https://github.com/asyncapi/community/pull/1634).
+
+## AsyncAPI Bounty Program
+[The AsyncAPI Bounty Program Q2 is underway](https://github.com/orgs/asyncapi/projects/36/views/4?sliceBy%5Bvalue%5D=bounty%2F2025-Q2), and we have 17 issues filled into capacity and assigned. It's exciting to see the program grow and the maintainers fully participating.
+
+## TSC x Ambassadors
+We are thrilled to welcome three new [Technical Steering Committee](https://www.asyncapi.com/community/tsc) (TSC) members.
+
+<Profiles profiles={[
+{
+    name: 'Charles d'Avernas',
```

Comment:
```suggestion
    name: "Charles d'Avernas",
```

Reply:
@akshatnema what's the requested change for?

Reply:
Nevermind I see why 😅